components/builder-web/app/shared/statuspage/_statuspage-indicator.component.scss

```diff
@@ -0,0 +1,25 @@
+// The CSS for the colored dot that represents your current status
+$red:#e74c3c;
+$orange:#e67e22;
+$yellow:#f1c40f;
+$green:#2ecc71;
+
+.color-dot {
+  border-radius: 99px;
```

Comment:
Why 99%?

Reply:
Yes it can! I just copied in the CSS from the plugin, so I can clean that up.